lib/ui/imagedecorated.flow

```diff
@@ -391,6 +393,51 @@ makeImageTextBox(ib : ImageTextBox) -> Form {
 	]);
 }
 
+
+makeImageTextColumn(textCol : ImageTextColumn) -> Form {
+	heightB = make(0.);
+	topLeft = Point(textCol.x, textCol.y);
+	fontSize = extractStruct(textCol.label.style, FontSize(36.)).size;
+	textColor = extractStruct(textCol.label.style, Fill(0xFFFFFF)).color;
+	isItalicFont = isItalic(textCol.label.style);
+	isBoldFont = isBold(textCol.label.style);
+	backColor = extractStruct(textCol.style, Fill(0));
+	backOpacity = extractStruct(textCol.style, FillOpacity(0.5));
+
+	material2nativeForm(
+		makeMaterialManager([]),
```

Comment:
Generally, using additional MaterialManager is a bad practice, but in this case it should be fine. In Material below manager is needed only for MParagraph transformation, but since custom color and font are used, it can be ignored. 

Trying to duplicate the same element in Form would be harder.